src/CachedInjectorFactory.php

```
@@ -23,9 +21,13 @@ final class CachedInjectorFactory
     /**
      * @param non-empty-string           $scriptDir
      * @param callable(): AbstractModule $modules
+     * @param mixed                      $cache           Deprecated parameter (no longer used)
      * @param SavedSingletons            $savedSingletons
+     *
+     * @deprecated The $cache parameter is deprecated. doctrine/cache has been abandoned.
+     *             Pass null or omit this parameter.
      */
-    public static function getInstance(string $injectorId, string $scriptDir, callable $modules, ?CacheProvider $cache = null, array $savedSingletons = []): InjectorInterface
+    public static function getInstance(string $injectorId, string $scriptDir, callable $modules, mixed $cache = null, array $savedSingletons = []): InjectorInterface
```

PHPMD (2.15.0)

Avoid unused parameters such as '$cache'. (undefined)

(UnusedFormalParameter)